lib/markbind/lib/parser.js

```diff
@@ -284,10 +284,10 @@ Parser.prototype._parse = function (node, context, config) {
                     || utils.fileExists(calculateBoilerplateFilePath(element.attribs.boilerplate,
                                                                      element.attribs.src, config));
     if (fileExists) {
-      const { src, fragment } = element.attribs;
+      const { src } = element.attribs;
       const resultDir = path.dirname(path.join('{{hostBaseUrl}}', path.relative(config.rootPath, src)));
       const resultPath = path.join(resultDir, utils.setExtension(path.basename(src), '._include_.html'));
-      element.attribs.src = fragment ? `${resultPath}#${fragment}` : resultPath;
+      element.attribs.src = resultPath;
```

Comment:
There is a bug in the rendering of panels with fragments after this line of code is changed. This suggests that the fragment is necessary after all:

<img src="https://user-images.githubusercontent.com/3168908/43059172-d14d3a1a-8e7d-11e8-8f3f-9b023a278277.png" width="450" />

Reply:
@nusjzx Btw this bug still exist, I presume you are not done with the PR, right? :P